homeassistant/components/wemo/switch.py

```diff
@@ -129,9 +143,10 @@ def detail_state(self):
             if standby_state == WEMO_STANDBY:
                 return STATE_STANDBY
             return STATE_UNKNOWN
+        return ""
```

Comment:
```suggestion
        return ""
```

I think you can `assert False` since this should be unreachable

Reply:
Thank you. Done. 21f3ea8